src/odemis/acq/test/stream_test.py

```diff
@@ -3292,12 +3309,130 @@ def test_spec_calib(self):
 
         specs.background.value = bckg
 
+        time.sleep(0.5)
+
         # Control spatial spectrum
-        im2d = specs.image.value
+        im2d = proj_spatial.image.value
         # Check it's a RGB DataArray
         self.assertEqual(im2d.shape, spec.shape[-2:] + (3,))
         self.assertTrue(numpy.any(im2d != prev_im2d))
 
+    def _create_temporal_spec_data(self):
+        # Temporal Spectrum
+        data = numpy.random.randint(1, 100, size=(251, 100, 1, 200, 300), dtype="uint16")
```

Comment:
A more typical _big_ shape would be (512, 512, 1, 50, 50). That's a > 1Gb in memory. Here, to keep the test from using too much memory, you could simulate a smaller acquisition, maybe (256, 128, 1, 20, 30). That's only 40Mb.